serializations.md

```diff
@@ -722,7 +741,19 @@ NOTE: As the POI contains the Merkle Patricia Tree nodes (e.g. not only their ha
 
 #### Micro Body
 ```
-[ <transactions> :: [binary()]
+[ <transactions>    :: [binary()]
+, <proof_of_fraud>> :: [binary()]
```

Comment:
I see this section is referred to by MSG_MICRO_BLOCK P2P message so that P2P message description seems up to date. Ok.